server/config.ts

```diff
@@ -44,6 +44,26 @@ export default {
     secret: get('SESSION_SECRET', 'app-insecure-default-session', requiredInProduction),
     expiryMinutes: Number(get('WEB_SESSION_TIMEOUT_IN_MINUTES', 120)),
   },
+  sqs: {
+    prisonEvents: {
+      accessKeyId: get('SQS_PRISON_EVENTS_ACCESS_KEY_ID', '', requiredInProduction),
+      secretAccessKey: get('SQS_PRISON_EVENTS_SECRET_ACCESS_KEY', '', requiredInProduction),
+      queueUrl: get(
```

Comment:
We could default these to the same values used for localstack - just any value, like foo and bar? Or set in .env.

Reply:
Access key and secret are not needed for localstack, but can stick foo and bar in. No need for anything though